pyquil/paulis.py

```diff
@@ -923,18 +923,18 @@ def exponential_map(term: PauliTerm) -> Callable[[float], Program]:
     coeff = term.coefficient.real
```

Comment:
linter is complaining about unused `ceoff` variable.

also unused imports in this file for the `X` and `PHASE` gates on line 48, above.

https://travis-ci.org/rigetti/pyquil/jobs/655073208?utm_medium=notification&utm_source=github_status

Reply:
The CI keeps failing. Then I find the tool, install locally, run tool, push, repeat. I hope this is the last one.

Reply:
you can run `make check-all` fyi to run all the "extra" checks (minus the test suite)